docs/v0.4/invokers/java.md

```diff
@@ -107,28 +107,30 @@ public class Hello implements Function<String, String> {
 
 Just as for Spring Boot based functions you can build your plain Java function either from local source or from source committed to a GitHub repository. Here we will only show the build from the GitHub repo:
 
-```bash
+```terminal
 riff function create hello --git-repo https://github.com/projectriff-samples/java-hello.git --handler functions.Hello
 ```
 
 The `--handler` option is the fully qualified name of the class that provides the function implementation.
 
 ## Deploying and invoking the function
 
-To deploy your function you need to select a runtime. The two options currently available are `core` and `knative` and we will select `knative`for this example:
+To deploy your function you need to select a runtime. The two options currently available are `core` and `knative` and we will select `core` for this example:
```

Comment:
rm this whole section and link to the core runtime docs. There's nothing special about invoking a java function.